### ISSUE TITLE
now able to preview notification content on ios

### ISSUE THREAD

**Issue by emilvincentz** [acterglobal/a3#1615]

**Comment by gnunicorn** [comment]
what does "preview" mean here?

**Comment by emilvincentz** [comment]
that you can see the content of the message if its a chat, or the image if its update.

**Comment by emilvincentz** [comment]
every message shows up like this.
![IMG_0368](https://github.com/acterglobal/a3/assets/77483554/ee5b8296-05e2-48ce-8281-92abea087a93)

**Comment by emilvincentz** [comment]
However to my surprise, I see for the first time that if I go back and view older notifications, I can preview (some) of them. 
![IMG_0369](https://github.com/acterglobal/a3/assets/77483554/160355e0-37ff-47ae-8dd5-d5aae0034b82)

**Comment by gnunicorn** [comment]
Yeah, that's Alan iOD issue that is blocked by sliding sync support, I am afraid 😟 ...

**Comment by emilvincentz** [comment]
Ah okey. But we aim to have sliding sync with the uniffi this quarter right?

**Comment by gnunicorn** [comment]
_after_ we have unffi, we can _start_ to think about doing sliding sync. but not this quarter, probably not, no.

**Comment by gnunicorn** [comment]
moving to backlog as this isn't something we can tackle yet...

**Comment by gnunicorn** [comment]
related to object notifications: https://github.com/acterglobal/a3/issues/2501
